demisto_sdk/commands/split/tests/ymlsplitter_test.py

```diff
@@ -266,6 +266,7 @@ def test_extract_code(tmpdir):
         assert "from CommonServerPython import *  #" in file_data
         assert file_data[-1] == "\n"
         assert "register_module_line" not in file_data
+        assert "### pack version 1.0.3" not in file_data
```

Comment:
add an assertion that "pack version 1.0.3" is in the original file, to make sure this nobody deletes it by mistake